tg/controllers/restcontroller.py

```diff
@@ -68,5 +68,9 @@ class RestController(DecoratedController, CoreDispatcher, RestDispatcher):
       `CrudRestController <../main/Extensions/Crud/index.html>`_  A way to integrate ToscaWdiget Functionality with RESTful Dispatch.
 
     """
+    @expose()
+    def options(self):
```

Comment:
Also please add a docstring that states why this is here (like "here only to prevent 404 in CORS preflight requests, override it in your own controller to properly manage CORS")